app/Database/Database.hs

```diff
@@ -16,18 +16,26 @@ import Database.Tables
 import WebParsing.ParseAll (parseAll)
 import Database.CourseVideoSeed (seedVideos)
 import Config (databasePath)
+import System.Directory (createDirectoryIfMissing)
+import Data.Text as T (length, findIndex, take, unpack)
+import Data.Maybe (fromMaybe)
 
 -- | Main function for setting up the database with course information.
 --
 -- TODO: Probably combine seeding of Distribution and Breadth tables,
 -- and split off from @parseAll@.
 setupDatabase :: IO ()
-setupDatabase = do setupDistributionTable
-                   print "Distribution table set up"
-                   setupBreadthTable
-                   print "breadth table set up"
-                   parseAll
-                   seedVideos
+setupDatabase = do
+    -- Create db folder if it doesn't exist
+    let ind = fromMaybe (T.length databasePath) (T.findIndex (=='/') databasePath)
+        db = T.unpack $ T.take ind databasePath
+    createDirectoryIfMissing False db
```

Comment:
It doesn't matter for this case, but if you change the config file to have the path `db/some/other/folders/` then we want the entire path to be created.

But now I realize that your current algorithm won't actually do that anyway (it seems to look for the first occurrence of a `/` rather than the last), so that should also be updated.